edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest/client/KubernetesClient.cs

```diff
@@ -48,5 +55,21 @@ public static async Task<Option<T>> WaitUntilAsync<T>(Func<Task<T>> action, Func
 
             return Option.None<T>();
         }
+
+        internal Task<V1DeploymentList> ListDeployments(string deviceSelector) => this.Kubernetes.ListNamespacedDeploymentAsync(this.DeviceNamespace, labelSelector: deviceSelector);
```

Comment:
My initial design decisions were based on an assumption that there will be one KubernetesClient that has only basic methods, and all the rest helper methods will go to extension classes: ServiceAccount, Deployment, PVC etc. Something has changed?